qcodes/instrument_drivers/Thorlabs/MFF10x.py

```diff
@@ -0,0 +1,52 @@
+from qcodes import Instrument
+
+
+class Thorlabs_MFF10x(Instrument):
+    """
+    Instrument driver for the Thorlabs MFF10x mirror flipper.
+    Args:
+        name (str): Instrument name.
+        device_id (int): ID for the desired mirror flipper.
+        apt (Thorlabs_APT): Thorlabs APT server.
+    Attributes:
+        apt (Thorlabs_APT): Thorlabs APT server.
+        serial_number (int): Serial number of the mirror flipper.
+        model (str): Model description.
+        version (str): Firmware version.
+    """
+
+    def __init__(self, name, device_id, apt, **kwargs):
+
+        super().__init__(name, **kwargs)
+
+        # save APT server link
+        self.apt = apt
+
+        # initialization
+        self.serial_number = self.apt.get_hw_serial_num_ex(48, device_id)
+        self.apt.init_hw_device(self.serial_number)
+        self.model = self.apt.get_hw_info(self.serial_number)[0].decode("utf-8")
+        self.version = self.apt.get_hw_info(self.serial_number)[1].decode("utf-8")
```

Comment:
wouldn't it be better to move the "decode" calls to `get_hw_info` method? the these two lines may become one:
```
self.model, self.version, _ = self.apt.get_hw_info(self.serial_number)
```